src/libcsg/modules/io/pdbreader.h

```diff
@@ -23,39 +23,45 @@
 #include <fstream>
 #include <votca/csg/topologyreader.h>
 #include <votca/csg/trajectoryreader.h>
+#include <votca/csg/atomtable.h>
 
 namespace votca { namespace csg {
 using namespace votca::tools;
 
 using namespace std;
 
 /**
-    \brief class for reading pdb files
+    brief class for reading pdb files
 
-    This class provides the TrajectoryReader + Topology reader interface
+    This class provides the Trajectory and Topology reader interface
     for pdb files
 
 */
 class PDBReader :
-    public TrajectoryReader, public TopologyReader
+    public TopologyReader , public TrajectoryReader, public AtomTable
 {
     public:
+
+        /// Constuctor
         PDBReader() {}
+        /// Destructor
         ~PDBReader() {}
-
         /// open a topology file
         bool ReadTopology(string file, Topology &top);
-
-        /// open a trejectory file
+        /// open a trajectory file
         bool Open(const string &file);
         /// read in the first frame
         bool FirstFrame(Topology &top);
         /// read in the next frame
         bool NextFrame(Topology &top);
+        /// set the topology file name: <name>.pdb (convention: ".pdb"="FIELD")
+        void setFname(string name) { _fname = name; return; }
+        /// get the topology file name: <name>.pdb (convention: ".pdb"="FIELD")
+        string getFname()          { return _fname; }
 
         void Close();
-
     private:
+        string _fname;
```

Comment:
where do you use ```_fname``` ?

Reply:
Other than in the setter and getter in the header file it is not used. Should I remove it then? I'm not sure if this is used elsewhere or not.